### PR TITLE
feat(shared-docs): Create @angular/docs package

### PR DESCRIPTION
**Pull request by josephperrott**
Create initial shared docs package

### CODE REVIEW

**Review by jelbourn — APPROVED** [review]
LGTM

File: docs/services/table-of-contents-scroll-spy.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/*!
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {TestBed, discardPeriodicTasks, fakeAsync, tick} from '@angular/core/testing';
```
Comment (devversion) [inline docs/services/table-of-contents-scroll-spy.service.spec.ts:9]: are we not running these specs? I worry we'll forget doing this in the future, if we don't do it now?

Comment (josephperrott) [reply docs/services/table-of-contents-scroll-spy.service.spec.ts:9]: I knew there was something on my todo list for this branch...

Comment (devversion) [reply docs/services/table-of-contents-scroll-spy.service.spec.ts:9]: Thanks! can you also provide additional context on the reason for this in the commit message? would be good to capture this just for the future (and maybe for me :D )

Comment (josephperrott) [reply docs/services/table-of-contents-scroll-spy.service.spec.ts:9]: Done.

File: docs/services/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    srcs = [
+        "index.ts",
+    ],
+    visibility = ["//docs:__subpackages__"],
+    deps = [
+        ":lib",
+    ],
+)
+
+ts_library(
```
Comment (devversion) [inline docs/services/BUILD.bazel:14]: The specs in here are not configured.

File: docs/utils/BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+ts_library(
+    name = "testing-helper",
+    srcs = [
+        "testing-helper.ts",
+    ],
+    visibility = ["//docs:__subpackages__"],
+)
+
+ts_library(
```
Comment (devversion) [inline docs/utils/BUILD.bazel:22]: The specs here are not configured.

**Review by devversion — APPROVED** [review]
LGTM overall, but need to be sure we are not having unused spec files

File: docs/utils/BUILD.bazel (hunk 3 above)
Comment (josephperrott) [reply docs/utils/BUILD.bazel:23]: turns out the spec file that was here wasn't used or a spec file.

**Comment by josephperrott** [comment]
This PR was merged into the repository by commit b09daf3df3011515795db63d7c8206b7ce9ee3b6.

**Pull request closed without merging**